x-pack/plugins/security_solution/common/api/detection_engine/index_management/read_privileges/read_privileges.schema.yaml

```diff
@@ -0,0 +1,43 @@
+openapi: 3.0.0
+info:
+  title: Read privileges API endpoint
+  version: '2023-10-31'
+paths:
+  /api/detection_engine/privileges:
+    get:
+      x-labels: [serverless, ess]
+      operationId: GetPrivileges
```

Comment:
Nit: I noticed a discrepancy in the naming between operation ids and file names and code. Operation ids are "get something" whereas in the code we have "read something". Why? Is "get something" a naming convention we're going to use for all our API endpoints in OAS?

Reply:
It's a good topic to tech time meeting. Since Detections API is most a RESTful API it focuses on data over actions on that data. It looks logical to have the same `Get`, `Post`, `Put`, `Delete` verbs instead of vague `Read`, `Create` and etc.

In this PR I place OpenAPI specs in existing folders to minimize the diff. I intend to perform renaming in a separate PR since it's a minor thing and won't impact functionality but the diff could be relatively large.